src/bilder/images/concourse/templates/vector.yaml

```diff
@@ -1,5 +1,11 @@
 ---
 sources:
+  vector_metrics:
```

Comment:
I've been debating whether it's worth capturing the Vector metrics. Happy to hear any thoughts that you have on the matter.

Reply:
Yeah, I think we just drop collection of those metrics from the agents. I think it would be helpful when running Vector as an aggregator, but having it on each host isn't terribly useful.